terraform/gcp/projects/meom-ige.tfvars

```diff
@@ -58,31 +58,35 @@ dask_nodes = {
     min : 0,
     max : 20,
     machine_type : "n1-standard-2",
-    labels: {}
+    labels: {},
   },
   "medium" : {
     min : 0,
     max : 20,
     machine_type : "n1-standard-8",
-    labels: {}
+    labels: {},
+    disk_type: "pd-ssd"
```

Comment:
just wanna confirm that you're intentionally making these `pd-ssd`, since you mentioned in another comment that you had missed some of them

Reply:
Good catch, @choldgraf! Fixed :)

Reply:
thanks @choldgraf and @yuvipanda -- will report back from the user base. Really appreciate this.

Reply:
@mmcky thanks for helping us make the service better for everyone!